dev/AnimatedVisualPlayer/AnimatedVisualPlayerAutomationPeer.h

```diff
@@ -8,7 +8,7 @@ class AnimatedVisualPlayerAutomationPeer :
     public ReferenceTracker<AnimatedVisualPlayerAutomationPeer, winrt::implementation::AnimatedVisualPlayerAutomationPeerT>
 {
 public:
-    AnimatedVisualPlayerAutomationPeer(winrt::AnimatedVisualPlayer const& owner);
+    explicit AnimatedVisualPlayerAutomationPeer(winrt::AnimatedVisualPlayer const& owner);
```

Comment:
>explicit [](start = 4, length = 8)

If people are calling this constructor any way we're doing it wrong. Making people put explicit here seems unnecessary